lib/collection/proxy-config.js

```diff
@@ -73,7 +59,7 @@ _.inherit((
              * The url mach for which the proxy has been associated with.
              * @type {String}
              */
-            match: MATCH_ALL_URLS,
+            match: new UrlMatchPattern(MATCH_ALL_URLS),
```

Comment:
Okay 👍 

Looks like `new RegExp()` matches all strings. So `new UrlMatchPattern()` should match all Urls.